agent/src/main/java/com/teamscale/jacoco/agent/AgentOptions.java

```diff
@@ -199,21 +200,25 @@ public String createJacocoAgentOptions() {
 	/** Sets output to none for normal mode and destination file in testwise coverage mode */
 	private String getModeSpecificOptions() {
 		if (useTestwiseCoverageMode()) {
-			DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd-HH-mm-ss.SSS", Locale.US);
-			return "sessionid=,destfile=" + new File(outputDirectory.toFile(),
-					"jacoco-" + dateFormat.format(new Date()) + ".exec").getAbsolutePath();
+			return "sessionid=,destfile=" + getTempFile("jacoco", "exec").getAbsolutePath();
 		} else {
 			return "output=none";
 		}
 	}
 
+	private File getTempFile(final String prefix, final String extension) {
+		DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd-HH-mm-ss.SSS", Locale.US);
```

Comment:
Why do we hard-code `Locale.US` here?

Reply:
Not sure why this ended up here. Removed it